src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/bootstrap_4_layout.html.twig

```diff
@@ -827,7 +827,7 @@
 
 {% block form_hint %}
   {% if hint %}
-    <div class="row col-lg hint-box">
+    <div class="hint-box">
```

Comment:
Just a comment about this one, we will stop using this bootstrap 4 layout soon. The new prestashop UI kit extends the layout from Symfony framework See here:
https://github.com/PrestaShop/PrestaShop/blob/83977178c98e95be1ca6b7393d16a40766feb8c5/src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit_base.html.twig#L48

Once all the forms will be migrated to the new UI kit form theme, we won't use this file at all, so I hope your modifications will still work as expected when everything is cleaned. Maybe we'll have to override some parts of the template to match your changes?

Reply:
@jolelievre Hi! Should not matter. Everything will work as expected. ;-)

Even if anybody uses or overrides only one file of my PR, or half, or everything, it has no effect on other files. Everything will look and function completely the same.